cart/cart.html

```diff
@@ -18,8 +18,8 @@
                     <li><a href="../index.html">Home</a></li>
                     <li><a href="../products.html">Products</a></li>
                     <li><a class="cart-link" href="#">Cart</a></li>
-                    <li><a href="wishlist.html">Wishlist</a></li>
-                    <li><a href="account.html">Account</a></li>
+                    <li><a href="#">Wishlist</a></li>
+                    <li><a href="../login page/login.html">Account</a></li>
```

Comment:
Try not to include blank spaces while folder naming.

Reply:
Yes, will avoid spaces from now on